### PR TITLE
Vet clinic database: query multiple tables

### PR DESCRIPTION
**Pull request by olawale-o**
## In this PR, I did the following:
- Create a table named **owners** with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - full_name: string
  - age: integer
- Create a table named **species** with the following columns:
  - id: integer (set it as autoincremented PRIMARY KEY)
  - name: string
- Modify **animals** table:
  - Make sure that id is set as autoincremented PRIMARY KEY
  - Remove column **species**
  - Add column **species_id** which is a foreign key referencing species table
  - Add column **owner_id** which is a foreign key referencing the owners table
- Modify your inserted animals to include owner information (owner_id):
  - Sam Smith owns Agumon.
  - Jennifer Orwell owns Gabumon and Pikachu.
  - Bob owns Devimon and Plantmon.
  - Melody Pond owns Charmander, Squirtle, and Blossom.
  - Dean Winchester owns Angemon and Boarmon.
- Write queries (using JOIN) to answer the following questions:
  - What animals belong to Melody Pond?
  - List of all animals that are pokemon (their type is Pokemon).
  - List all owners and their animals, remember to include those that don't own any animal.
  - How many animals are there per species?
  - List all Digimon owned by Jennifer Orwell.
  - List all animals owned by Dean Winchester that haven't tried to escape.
  - Who owns the most animals?

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -15,4 +15,37 @@ CREATE TABLE animals (
...
+-- Create a table named owners with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- full_name: string
+-- age: integer
+
+CREATE TABLE owners (
+  id INT UNIQUE GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  full_name VARCHAR(100),
+  age INT
+);
```
Comment (Davidosky007) [inline schema.sql]: * Kindly add a primary key that increases automatically as new records are created (ie PRIMARY KEY (id)) [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=the%20following%20columns%3A-,id%3A%20integer%20(set%20it%20as%20autoincremented%20PRIMARY%20KEY),-full_name%3A%20string)

File: schema.sql [hunk 2]
```diff
@@ -15,4 +15,37 @@ CREATE TABLE animals (
...
+
+-- Create a table named species with the following columns:
+-- id: integer (set it as autoincremented PRIMARY KEY)
+-- name: string
+
+CREATE TABLE species (
+  id INT UNIQUE GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+  name VARCHAR(100)
+);
```
Comment (Davidosky007) [inline schema.sql]: * Kindly add a primary key that increases automatically as new records are created (ie PRIMARY KEY (id)) [Check requirement](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_multiple_tables.md#:~:text=the%20following%20columns%3A-,id%3A%20integer%20(set%20it%20as%20autoincremented%20PRIMARY%20KEY),-full_name%3A%20string)

**Review by Davidosky007 — CHANGES_REQUESTED** [review]
## Hi @olawale-o 👋, 

Good job so far! 💪 
* Descriptive commit messages ✔️ 
* Well written PR ✔️ 
* Profesional readme file ✔️ 
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: schema.sql (hunk 2 above)
Comment (olawale-o) [reply schema.sql]: Thank you for pointing it out

File: schema.sql (hunk 1 above)
Comment (olawale-o) [reply schema.sql]: Thank you for pointing it out

**Review by arslanbisharat — APPROVED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashbaord. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Comment by olawale-o** [comment]
Hi @arslanbisharat 

Thank you for the time and effort on the review.

**Pull request merged**